tst/com/amazon/corretto/crypto/provider/test/TestUtil.java

```diff
@@ -60,10 +71,31 @@ public class TestUtil {
         new String[] {"secp256k1", "1.3.132.0.10"},
       };
 
-  public static final boolean isFips() {
+  public static boolean isFips() {
     return NATIVE_PROVIDER.isFips();
   }
 
+  public static byte[] intArrayToByteArray(final int[] array) {
+    final byte[] result = new byte[array.length];
+    for (int i = 0; i != array.length; i++) {
+      result[i] = (byte) array[i];
```

Comment:
is this safe? i wonder if we should throw an exception if the array element is >255

Reply:
I was copy-pasting tests from LC and since bytes are signed in Java, for values greater than 127 (or 128), Java wanted the hex values to be typecast to bytes. So I decided to do the copy-pasting as int and do the casting in this method. Having said that, I added the bound checks to this method, despite being only used for testing.